pkg/check/datastore.go

```diff
@@ -27,8 +30,48 @@ const (
 	dataCenterType        = "Datacenter"
 	DatastoreInfoProperty = "info"
 	SummaryProperty       = "summary"
+
+	dataStoreType = "type"
+)
+
+var (
+	dataStoreTypesMetric = metrics.NewGaugeVec(
```

Comment:
Did we really wanted this to be a gauge?

Reply:
All other metrics are gauges.